src/index.html

```diff
@@ -0,0 +1,161 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Weather project</title>
+        <link rel="stylesheet"
+            href="https://fonts.googleapis.com/css?family=Ubuntu">
+
+        <link rel="stylesheet" type="text/css" href="index_style.css">
+
+    </head>
+    <body>
+        <div class="headerimg">
+            <img alt="leftcloudimg" class="leftcloud" src="../Assets/HTML&CSS/cloud-8121.png">
+            <img alt="rightcloudimg" class="rightcloud" src="../Assets/HTML&CSS/cloud-8121.png">
+            <div class="topsegment">
+                <div id="topfront">
+                    <div id="topimg1">
+                        <img id="topimg" src="../Assets/HTML&CSS/Icons_for_cities/maseru.svg" alt="x">
+                    </div>
+
+                    <div id="cityselect" style="position: relative;">
+                        <select id="optionselect">
+                            <option>New Delhi</option>
+                            <option>New abi</option>
+                        </select>
+                    </div>
+                    <div>
+
+                    </div>
+                    <div id="rightdown">
+                        <div id="time" class="a">
+                            <h6 id="time-text">
+                                10:10: <small> 44 </small>
+                            </h6>
+                            <img alt="amimg" id="time-img" src="../Assets/HTML&CSS/GeneralImages&Icons/amState.svg">
+                        </div>
+                        <div id="date" class="a">
+                            <h6 id="date-text">01-MAR-2022</h6>
+                        </div>
+                    </div>
+                </div>
+                <div id="topmiddle">
```

Comment:
Usage of ids can be minimized as id has more specificity than class. Also, id is unique and can't have the same names for two tags. 
Whereas class names help in reusing the css styles.